imap_processing/cdf/config/imap_ultra_global_cdf_attrs.yaml

```
@@ -273,8 +273,8 @@ imap_ultra_l1a_90sensor-imgparams:
     Logical_source_description: IMAP-Ultra Instrument Level-1A Image Parameters Data for Ultra90.
 
 # Ultra L2 Global attributes for different sensors and sky tiling types and durations
-imap_ultra_l2_enamap-hf:
+imap_ultra_l2_enamap:
     <<: *instrument_base
-    Data_type: L2_{sensor}Sensor-enamap{tiling}-hf-{duration}>Level-2 ENA Intensity Map for Ultra{sensor}
-    Logical_source: imap_ultra_l2_{sensor}sensor-enamap{tiling}-hf-{duration}
-    Logical_source_description: IMAP-Ultra Instrument Level-2 ENA Intensity Map Data for Ultra{sensor} on {tiling} tiling in heliospheric frame over {duration} duration.
+    Data_type: L2_u{sensor}-ena-h-{inertial_frame_short_name}-nsp-full-hae-{resolution_string}-{duration}>Level-2 ENA Intensity Map for Ultra{sensor}
```

```suggestion
    Data_type: L2_u{sensor}-ena-h-{inertial_frame_short_name}-nsp-full-hae-{resolution_string}-{duration}>Level-2 Sensor ENA Intensity Map for Ultra{sensor}
```
Maybe add the actual word sensor to the long name?

---

Oh nvm I see that you removed it from the short name above.